source/installing.rst

```diff
@@ -19,39 +19,40 @@ Like many other |percona| products, |pdp| is installed by using the
    <https://www.percona.com/doc/percona-repo-config/percona-release.html#percona-release-update-latest-version>`_
    of |percona-release| for details.
 
-As soon as |percona-release| is up-to-date, *set up* the |pdp| product (``ppg-11``).
+As soon as |percona-release| is up-to-date, *set up* the required version of the
+|pdp| product (``ppg-<version>``). For example, to set up |pdp| 12.2, use the following command:
 
 .. code-block:: bash
 
-   $ sudo percona-release setup ppg11
+   $ sudo percona-release setup ppg12.2
```

Comment:
there are 2 ways to call setup with and without dash
percona-release setup ppg12.2 or percona-release setup ppg-12.2
as for me it is better to use variant with dash